package.json

```diff
@@ -21,10 +21,13 @@
   },
   "homepage": "https://github.com/eddiejaoude/EddieBot#readme",
   "dependencies": {
-    "discord.js": "^12.1.1"
+    "discord.js": "^12.1.1",
+    "pino": "^6.3.2",
+    "pino-pretty": "^4.0.0"
```

Comment:
Is this dependency being used? I checked the [docs](https://github.com/pinojs/pino/blob/afb639c79a1135790e9302b94f054a6118bffa6f/docs/pretty.md#api-example) and maybe we could use it similar to this:
```ts
const log = pino({
  prettyPrint: {
    levelFirst: true
  },
  prettifier: require('pino-pretty')
})
```

Reply:
From what I understand from the docs, the above is equal to... (what do you think?)

```typescript
// Note: the default prettifier module is pino-pretty, so the preceding example could be:
const pino = require('pino')
const log = pino({
  prettyPrint: {
    levelFirst: true
  }
})
```

<img width="948" alt="Screenshot 2020-06-11 10 25 56" src="https://user-images.githubusercontent.com/624760/84368808-f416c600-abcd-11ea-90ff-6bff57453f24.png">

Reply:
Yes, I didn't see the default 😅

Reply:
No worries, its good to double check 👍